### PR TITLE
User login logout

### PR DESCRIPTION
**Pull request by siddheshbarhate31**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,44 @@
...
+
+We can add the tables in the database in two methods:
+
+1-By running these commands: 
+
+    from app import db;
+    from app.model import user;
+    from app.model import bank_account;
+    from app.mddel import account_transaction_details;
+But, if we want to add new column in the table we have to drop all the tables and the data is also lost.
```
Comment (shreyas-damle) [inline README.md]: Remove this, as we already have commands to do this.

File: README.md [hunk 2]
```diff
@@ -1 +1,44 @@
...
+
+2.Using the flask-script and flask-migrate:
+
+###Installation of flask-script and flask-migrate:
+     pip install flask-script;
+     pip install flask-migrate;
+
+Import these in the init.py and run.py and it handles the database migration by running the migration commands.
```
Comment (shreyas-damle) [inline README.md]: Remove this step and mention about virtual environment and python version and installing these things from requirements.txt

File: README.md [hunk 3]
```diff
@@ -1 +1,44 @@
...
+Upgrade to a later version
+
+##Use of flask-script and flask-migrate:
+- Using these data migration is done and we can add new column in the table using migration commands without losing any data
+- And, can also create table or update it without dropping all the tables.
+
+##-Have made the collection of APIs in postman
+
+
+ https://www.getpostman.com/collections/c49ee49f5cf73a8b014a
```
Comment (shreyas-damle) [inline README.md]: Instead of adding postman link, please add the postman collection in the git repo.

File: app/Schema/account_transaction_details_schema.py [hunk 4]
```diff
@@ -48,16 +47,15 @@ class FundTransferSchema(ma.Schema):
 
     """Adding the Schema validations to the FundTransfer """
 
-    from_account = fields.Str(required=True, validate=Regexp("^[0-9]{10}$|^[0-9]{12}$"))
-    to_account = fields.Str(validate=Regexp("^[0-9]{10}$|^[0-9]{12}$"))
-    transaction_amount = fields.Int(required=True)
+    from_account = fields.Str(required=True, validate=(Regexp("^[0-9]{10}$|^[0-9]{12}$"), Length(equal=10)))
+    to_account = fields.Str(validate=(Regexp("^[0-9]{10}$|^[0-9]{12}$"), Length(equal=10)))
+    transaction_amount = fields.Int(strict=True, required=True, validate=Range(min=1000, max=50000))
```
Comment (shreyas-damle) [inline app/Schema/account_transaction_details_schema.py]: Why these changes are coming now? We have done those almost a month ago.

File: migrations/env.py [hunk 5]
```diff
@@ -0,0 +1,93 @@
+from __future__ import with_statement
```
Comment (shreyas-damle) [inline migrations/env.py:1]: Do we need migrations folder in the git repository? Please confirm.

Comment (siddheshbarhate31) [reply migrations/env.py:1]: Yes, we need the migration folder because we are manually importing the app.models in migrations/env.py

File: README.md (hunk 1 above)
Comment (siddheshbarhate31) [reply README.md]: yes, have removed it as suggested.

File: app/Schema/account_transaction_details_schema.py (hunk 4 above)
Comment (siddheshbarhate31) [reply app/Schema/account_transaction_details_schema.py]: Have not changed it now, it have been done before

File: README.md (hunk 2 above)
Comment (siddheshbarhate31) [reply README.md]: Added the version, installations and virtual env in the readme file

File: README.md (hunk 3 above)
Comment (siddheshbarhate31) [reply README.md]: Added the postman collection json file in the git repo.
